packages/reactDom/src/ReactDOM.ml

```diff
@@ -18,6 +18,8 @@ let attribute_to_html attr =
   (* true attributes render solely the attribute name *)
   | Bool (name, _, true) -> Html.present name
   | Style styles -> Html.attribute "style" (ReactDOMStyle.to_string styles)
+  (* ActionFunction should be always with empty value on the markup *)
```

Comment:
This comment doesn't add much, I would rather add comments that explain the why, not the how. Looking at L22 you can already guess that the output if empty.